docs/en/reference/query-builder.rst

```diff
@@ -369,3 +369,26 @@ in your query as a return value:
         ->where('email = ' .  $queryBuilder->createPositionalParameter($userInputEmail))
     ;
     // SELECT id, name FROM users WHERE email = ?
+
+Caching
+-------
+
+To use the result cache, it is necessary to call the method
+``enableResultCache($cacheProfile)`` and pass a instance of
+``Doctrine\DBAL\Cache\QueryCacheProfile`` with lifetime in seconds.
+Optionally can be specified cache key.
+
+.. code-block:: php
+
+    <?php
+
+    $queryBuilder
+        ->select('id', 'name')
+        ->from('users')
+        ->enableResultCache(new QueryCacheProfile(300, 'some-key'))
+    ;
+
+.. note::
+
+    For more details and how to configure the result cache check caching section in
+    documentation.
```

Comment:
Can you please link to the caching section of the docs?

Reply:
Can somebody help me how to add link there. I tried ``` `caching section in
    documentation <./caching.rst>` ``` and ``` `caching section in
    documentation <https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/caching.html>` ```. I don't like absolute path. It is posible use realtive link there? I don't know how it works. RST files are for me new thing.

Reply:
From [StackOverflow](https://stackoverflow.com/a/66289376/146187):

> An existing file {example.rst} may be linked to with the following syntax:
> ```
> :ref:`Optional Link text <example>`
> ```
See the following bits for example: https://github.com/doctrine/dbal/blob/14675c0436fcd65b3727db74e691412ee4930622/docs/en/reference/data-retrieval-and-manipulation.rst#L344 https://github.com/doctrine/dbal/blob/7b24c55dc81231fb8265dcc0d0505c19e5753472/docs/en/reference/types.rst#L481

Reply:
@SenseException I'd leave the exercise of linking documentation articles to the maintainers. It's not trivial and there's no way to check the correctness of their change for a first-time contributor.

Reply:
I put ``` :ref:`Caching <caching>` ``` there, hopefully it will work. Thanks.